app/controllers/lookup_keys_controller.rb

```diff
@@ -1,13 +1,10 @@
+# Base class for all LookupKeys descendants controllers
+# The index method needs to be always implemented in the subclass
```

Comment:
should all the non-implemented methods raise `NotImplemented` to make it clear? it isn't just index, there are a few others that are overriden in the children.

Reply:
well the index is necessary though, right?
I don't think it should be `NotImplemented` - http://chrisstump.online/2016/03/23/stop-abusing-notimplementederror/

Reply:
fair enough.

Reply:
> well the index is necessary though, right?
> I don't think it should be `NotImplemented` - http://chrisstump.online/2016/03/23/stop-abusing-notimplementederror/

+1, I read the same post when I was thinking about adding `NotImplemented`